net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch

```diff
@@ -2,7 +2,7 @@ diff --git a/Makefile.am b/Makefile.am
 index 22647be..7dcf0de 100644
 --- a/Makefile.am
 +++ b/Makefile.am
-@@ -32,7 +32,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
+@@ -45,7 +45,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
```

Comment:
same here. what was changed?

Reply:
Patch was refreshed.

Reply:
Did you force-push the branch? I still see the 'old' diff.

Reply:
The only change was the line numbers.